deploy/packages/bbp-packages.yaml

```diff
@@ -128,6 +128,7 @@ packages:
       - py-bluepy@0.14.14
       - py-bluepymm@0.7.49 ^neuron%intel@19.0.4
       - py-bluepyopt@1.9.27 ^neuron%intel@19.0.4
+      - py-currentscape@0.0.1
```

Comment:
It's listed as `0.0.1.dev0` below…

Reply:
Hello!
I've been added to the GH BlueBrain organization, and I changed the version to 0.0.1, but the BlueBrain Deployment Test Build does not seem to launch. Is there something I'm missing?